runtime/codert_vm/decomp.cpp

```diff
@@ -113,7 +118,8 @@ static J9OSRFrame* findOSRFrameAtInlineDepth(J9OSRBuffer *osrBuffer, UDATA inlin
 static void   jitFramePopNotificationAdded(J9VMThread * currentThread, J9StackWalkState * walkState, UDATA inlineDepth);
 static void decompPrintMethod(J9VMThread * currentThread, J9Method * method);
 static UDATA decompileAllFrameIterator(J9VMThread * currentThread, J9StackWalkState * walkState);
-static J9JITDecompilationInfo * deleteDecompilationForExistingFrame(J9VMThread * decompileThread, J9JITDecompilationInfo * info);
+static J9JITDecompilationInfo * deleteDecompilationForExistingFrame(J9VMThread *currentThread, J9VMThread *decompileThread, J9JITDecompilationInfo *info);
+static J9JITDecompilationInfo * addDecompilationHelper(J9VMThread * currentThread, J9StackWalkState * walkState, UDATA reason, J9JITDecompilationInfo **previous);
```

Comment:
in the PR changes, multiple formats are used: i) `J9Method *method` vs. `J9Method * method` and ii) `(void *)` vs `(void*)`; for consistency, one format should be adopted. 
```suggestion
static J9JITDecompilationInfo *deleteDecompilationForExistingFrame(J9VMThread *currentThread, J9VMThread *decompileThread, J9JITDecompilationInfo *info);
static J9JITDecompilationInfo *addDecompilationHelper(J9VMThread *currentThread, J9StackWalkState *walkState, UDATA reason, J9JITDecompilationInfo **previous);
```